src/ctia/entity/sighting/es_store.clj

```diff
@@ -36,7 +36,9 @@
       :internal em/boolean-type
       :observables em/observable
       :observables_hash em/token
-      :relations em/observed-relation})}})
+      :relations em/observed-relation
+      :context {:type "object"
+                :enabled false}})}})
```

Comment:
Dynamic being false at the top level of the entity, it should not be indexed if you do not define this mapping.